packages/react-dom-bindings/src/client/DOMPropertyOperations.js

```diff
@@ -134,6 +124,11 @@ export function getValueForProperty(
     }
 
     // shouldRemoveAttribute
+    switch (typeof expected) {
+      case 'function':
+      case 'symbol': // eslint-disable-line
+        return value;
+    }
```

Comment:
I don't follow this part.

Reply:
I think this was a bug earlier. If you have an attribute but the prop value was function or symbol, then it's an error because there shouldn't exist an attribute at all.

Not sure why it's not covered by attribute table.

Returning the prop value causes the error because it won't be the same as expected.